packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1618,7 +1629,7 @@ class ClipboardStatusNotifier extends ValueNotifier<ClipboardStatus> with Widget
   }) : super(value);
 
   bool _disposed = false;
-  /// True iff this instance has been disposed.
+  /// True if this instance has been disposed.
```

Comment:
Yes in this context, the "if and only if" is implied (and it's empty prose), so "if" is better anyhow.